iconservice/iiss/ipc/reward_calc_proxy.py

```diff
@@ -35,23 +36,27 @@ class RewardCalcProxy(object):
 
     """
 
-    def __init__(self):
+    IPC_TIMEOUT = 0.1
+
+    def __init__(self, calc_callback: Callable[[tuple], Any] = None):
         Logger.debug(tag=_TAG, msg="__init__() start")
 
         self._loop = None
         self._message_queue = None
         self._ipc_server = IPCServer()
         self._message_queue: Optional['MessageQueue'] = None
-        self._calculation_result: Optional[tuple] = None
         self._reward_calc: Optional[Popen] = None
+        self._calculation_callback: Optional[Callable] = calc_callback
 
         Logger.debug(tag=_TAG, msg="__init__() end")
 
     def open(self, sock_path: str, iiss_db_path: str):
         Logger.debug(tag=_TAG, msg="open() start")
 
         self._loop = asyncio.get_event_loop()
-        self._message_queue = MessageQueue(self._loop)
+        self._message_queue = MessageQueue(loop=self._loop,
+                                           notify_message=VersionResponse,
```

Comment:
I'm not sure it is okey to ignore type mismatch. becuase that param's type is tuple, but you put the response instance.

Reply:
how about the changes below?
```suggestion
                                           notify_message=(VersionResponse,),
```

Reply:
okay, reasonable